### ISSUE TITLE
Bugs in CMI Portals

### ISSUE THREAD

**Issue by CoreXMC** [Zrips/CMI#825]
CMI Portals

The online guide is saying to use a wooden hoe to record positions 1 / 2 to create a portal.
https://www.spigotmc.org/wiki/cmi-portals/
Nothing happens. So if you go to step 2 with the command /cmi portals new [Name], you get an error message saying that you need to use a wooden shovel instead.
The wooden shovel works to record positions but when you are going to the next step, you get an error message saying that you need to records positions with a wooden hoe.

Probably a simple error in the code.

**Cmi Version (using`/cmi version`):** 
CMI8.0.8.1
**Server Type (Spigot/Paperspigot/etc):** 
Spigot
**Server Version (using `/ver`):** 
Spigot 1.12

**Comment by Zrips** [comment]
Will update wiki page. Tho, about which next step are you talking about? After selecting 2 points with left and right buttons, you simply performing command /cmi portals new [name] and that's that, you have a portal and GUI opens automatically to edit it.

**Comment by CoreXMC** [comment]
I was stuck before the command part. The problem was with selecting/recording the positions correctly to be able to use the command after. Something was interfering between positions recording and CMI portals.

I figured out where the problem was on my side. I removed WorldEdit plugin and CMI portal worked just fine. Apparently, the WorldEdit version we are using break mostly all portals plugins we tried using in-game pos1/pos2 to set positions for portals.
